test/e2e/README.md

```diff
@@ -49,3 +49,21 @@ For the ginkgo-based test, normally every test case is run. However, the script
 The kubestellar controller-manager will be invoked with `-v=2` unless otherwise specified on the command line with `--kubestellar-controller-manager-verbosity $number`. This verbosity can not be set to a value other than 2 when using `--released`.
 
 The transport controller will be invoked with `-v=4` unless otherwise specified on the command line with `--transport-controller-verbosity $number`.
+
+# Compatibility Matrix Testing
+
+A robust E2E compatibility matrix test is provided in `run-compat-matrix.sh`. This script:
```

Comment:
Robustness is the norm. Explicitly saying it sounds like bragging or making a big deal of ordinariness. I would omit this word.